WordPress/wordpress-playground
# Docs: Move Steps Shorthands to a separate page to fix Steps TOC

## What is this PR doing?

Moves the Steps Shorthands documentation, merged in https://github.com/WordPress/wordpress-playground/pull/1261, to a separate page.

## What problem is it solving?

@ironnysh [noted](https://github.com/WordPress/wordpress-playground/pull/1261#issuecomment-2060855003) that merging the PR messed up the table of contents on the steps page:

<img width="512" alt="323169294-76cb6e7b-2ec5-4f66-b6b6-04e0dbcd1edc" src="https://github.com/WordPress/wordpress-playground/assets/205419/2fa204c9-5652-40e1-8903-04feed74d686">

That specific ToC is generated using a custom React component that doesn't seems to handle merging new entries with any pre-existing structure. We could fix it, but since we might soon move to a [Playground-based documentation workflow](https://github.com/adamziel/playground-docs-workflow), investing time in Docusaurus setup doesn't make much sense. I went for the easy way out and moved the information to a new page.

## Testing Instructions

Run `nx dev docs-site` locally and confirm the toc on the Steps page is in tact.
